discovery_engine/lib/assets/asset_manifest.json

```diff
@@ -1,14 +1,20 @@
 {
   "assets": [
+    {
+      "id": "smbertConfig",
+      "url_suffix": "smbert_quantized_v0000/config.toml",
+      "checksum": "4cd20e13a53a69348c8ba9de085c3246de954aebd8cb4f3f90d50ea1ebed537c",
+      "fragments": []
+    },
     {
       "id": "smbertVocab",
-      "url_suffix": "smbert_v0002/vocab.txt",
+      "url_suffix": "smbert_quantized_v0000/vocab.txt",
       "checksum": "ffb2398810fa977ee39e8a0103b07f9c2614f1f9825df53a2c99e71569e3451c",
       "fragments": []
     },
     {
       "id": "smbertModel",
-      "url_suffix": "smbert_v0002/smbert-quantized.onnx",
+      "url_suffix": "smbert_quantized_v0000/model.onnx",
```

Comment:
why not keep the `v0002` suffix? more then that, I think we embed the manifest into the mobile app, so if we change the paths in S3 that might have some consequences for the mobile users, although probably not too many or maybe we don't care?

Reply:
> why not keep the v0002 suffix?

we never had an archive of this name, hence i started from `v0000`. but we can also start from a higher version. since it was split from the `smbert_v0002` and the `config.toml` was added, should we start from `v0003` then?

> more then that, I think we embed the manifest into the mobile app, so if we change the paths in S3 that might have some consequences for the mobile users, although probably not too many or maybe we don't care?

i was always under the impression, that the app doesn't deal with the details of the assets, but just fetches whatever is written in the asset manifest. in the previous asset version changes i also didn't change more than this. am i missing sth?

Reply:
> we never had an archive of this name, hence i started from v0000. but we can also start from a higher version. since it was split from the smbert_v0002 and the config.toml was added, should we start from v0003 then?

If that's a new version of smbert quantized model, then I would start with v0003. If this is a new model altogether then a v0000 should be good.

> i was always under the impression, that the app doesn't deal with the details of the assets, but just fetches whatever is written in the asset manifest. in the previous asset version changes i also didn't change more than this. am i missing sth?

I'm not opposed to changing asset manifest, I'm just worried if we can still support older versions of the manifest that are embedded on user devices (hence we can't change them, the app needs to be updated). So it's more about having references from old manifest still accessible on S3 to download, but it's maybe an edge case for user who installed the app and didn't fetch assets yet.

Reply:
> If that's a new version of smbert quantized model, then I would start with v0003. If this is a new model altogether then a v0000 should be good.

i'll change it to `v0003` then 👌 

> I'm not opposed to changing asset manifest, I'm just worried if we can still support older versions of the manifest that are embedded on user devices (hence we can't change them, the app needs to be updated). So it's more about having references from old manifest still accessible on S3 to download, but it's maybe an edge case for user who installed the app and didn't fetch assets yet.

this should still work, the buckets contain all the old model versions. if someone with an old asset manifest starts to fetch them, it will succeed.

Reply:
>this should still work, the buckets contain all the old model versions. if someone with an old asset manifest starts to fetch them, it will succeed.

this is exactly what I wanted to hear 👍